Dockerfile

```diff
@@ -1,7 +1,7 @@
 FROM node:10.15.3-alpine
 
-# Create app directory
-WORKDIR /usr/src/app

```

Comment:
This was changed in a previous PR, just wondering why it's changing again? There was some discussion about standardising on /usr/src/app previously.
https://github.com/DEFRA/mine-support/commit/86fefec4073beaaa99b6547e2b839bfaa73eb925

Reply:
I wasn't aware of that. I switched to a folder owned by the `node` user after dealing with a permission issue in the build pipeline but we can make it work with `/usr/src/app` if that's what we want to use.